### DISCUSSION TITLE
Help with parsing [BODY]

### DISCUSSION THREAD

**Discussion by xor777** [TwiN/gatus#708] [General]
My endpoint returns the following JSON:

{"error":null,"result":{"redis":true,"postgres":true,"rabbitMq":true}}

In the conditions check, I write the following:

conditions:
  - "[STATUS] == 200"
  - "[BODY].error == null"

But Gatus reports: [BODY].error (INVALID) == null

Could you guys help, what the problem might be?"

**Comment by TwiN** [comment]
Well since `error` is not set to any value (`null`), I'd imagine it's being treated as an empty value.

Have you tried doing `len([BODY].error) == 0` instead?

**Reply by xor777** [reply]
Tried, no success. Looks like its better idea to change error:null to something like result:0

**Reply by TwiN** [reply]
How about `has([BODY].error) == false`?
